src/scripts/templates/enlistment_edit.html

```diff
@@ -66,8 +66,9 @@ <h2>Enlistment</h2>
             {{#select timezone}}
             <option value="">Select...</option>
             <option value="EST">7pm EST (Midnight GMT)</option>
+            <option value="PST">6pm PST (2am GMT)</option>
             <option value="GMT">2pm EST (7pm GMT)</option>
-            <option value="Either">Either</option>
+            <option value="Any">Any</option>
```

Comment:
This value will be saved to the database. The `enlistments` table is an enum with values `enum('EST','GMT','Either','Neither','PST')`. This would therefore be saved as `NULL` unless we update the database column. @swomma is familiar with the use case here, so I defer to him on whether we should do that; but we'd need to before merging.